packages/database/src/migrations/20201110234830-DeleteOldLaosSchoolsTextbookData-modifies-data.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  await db.runSql(`
+    -- Drop triggers so that all the deletes don't flood the dhis sync queue
+    DROP TRIGGER IF EXISTS survey_response_trigger
```

Comment:
I would assume that this is not required since Laos surveys do not get synced to DHIS. Again, it's fine to leave this here (although if that's the case indeed, I would be tempted to remove it)

Reply:
It seems I reviewed this at the same time as @edmofro , whose comment explains why triggers are dropped